contracts/token/Vesting.sol

```diff
@@ -5,32 +5,39 @@ import "./SyndicateERC20.sol";
 import "../utils/Ownable.sol";
 
 contract Vesting is Ownable {
-    uint256 public startTime;
-    uint256 public vestingDays;
-    address public syn;
+  uint256 public startTime;
+  uint256 public vestingDays;
+  address public syn;
 
-    mapping(address => uint256) public grants;
+  mapping(address => uint256) public grants;
 
-    constructor(address _syn, address[] memory _receivers, uint256[] memory _awards) {
-        require(_awards.length == _receivers.length, "Vesting: length unmatch");
-        for (uint256 i = 0; i < _receivers.length; i++) {
-            grants[_receivers[i]] = _awards[i];
-        }
-        startTime = block.timestamp;
-        syn = _syn;
-        vestingDays = 365 + 31;
-    }
+  constructor(address _syn, uint256 _vestingDays) {
+    syn = _syn;
+    vestingDays = _vestingDays;
```

Comment:
why make it a parameter?

Reply:
Just in case we like to decide to change it at last minute